wandb/apis/public.py

```diff
@@ -2382,9 +2389,9 @@ def delete(self, delete_artifacts=False):
         """
         mutation = gql(
             """
-            mutation DeleteFromRunQueue($queueID: String!, $runQueueItemID: String!)
+            mutation DeleteFromRunQueue($queueID: ID!, $runQueueItemId: ID!)
```

Comment:
TODO: Change this query to using a NAME, rather than ID. All other references to "queue" are to the string: queue_name. (this will require both backend and frontend changes though)